tests/vmcheck/sync.sh

```diff
@@ -32,13 +32,23 @@ fi
 
 restorecon -v /usr/bin/{rpm-,}ostree /usr/libexec/rpm-ostreed
 
+overrides_dir=/etc/systemd/system/rpm-ostreed.service.d
+mkdir -p $overrides_dir
+
 # For our test suite at least, to catch things like
 # https://github.com/projectatomic/rpm-ostree/issues/826
-mkdir -p /etc/systemd/system/rpm-ostreed.service.d
-cat > /etc/systemd/system/rpm-ostreed.service.d/fatal-warnings.conf << EOF
+cat > $overrides_dir/fatal-warnings.conf << EOF
 [Service]
 Environment=G_DEBUG=fatal-warnings
 EOF
 
+# In the developer workflow, it's just not helpful to
+# have the daemon auto-exit. But let's keep it as a separate
+# override file to make it easy to drop if needed.
+cat > $overrides_dir/no-idle-exit.conf << EOF
```

Comment:
Mmm...by default?  Hmm.   Well, let's go with it for now.

Reply:
Yeah, we're diverging from what we ship a bit, but I think it makes the most sense for developers. E.g. it keeps the `HACKING.md` instructions as simple as possible. Note also that this only happens for `vmsync`, so our CI does not enable this.